openedx/core/djangoapps/course_live/tab.py

```diff
@@ -0,0 +1,35 @@
+"""
+Configurations to render Course Live Tab
+"""
+from django.utils.translation import gettext_lazy
+from lti_consumer.models import LtiConfiguration
+
+from common.lib.xmodule.xmodule.course_module import CourseBlock
+from common.lib.xmodule.xmodule.tabs import TabFragmentViewMixin
+from lms.djangoapps.courseware.tabs import EnrolledTab
+from openedx.core.djangoapps.course_live.models import CourseLiveConfiguration
+from openedx.features.lti_course_tab.tab import LtiCourseLaunchMixin
+
+
+class CourseLiveTab(LtiCourseLaunchMixin, TabFragmentViewMixin, EnrolledTab):
+    """
+    Course tab that loads the associated LTI-based live provider in a tab.
+    """
+    type = 'lti_live'
+    priority = 42
```

Comment:
What is the priority based on?

Reply:
According to this priority where exactly in the tab list will this tab appear?

Reply:
I have added screenshot in description and it will render after discussions tab